lib/inventories/permission.rb

```diff
@@ -0,0 +1,107 @@
+module Inventories
+  class Permission
+    PERMISSIONS = [:facilitator, :participant]
+
+    attr_reader :inventory
+    attr_reader :user
+
+    def initialize(inventory:, user:)
+      @inventory = inventory
+      @user = user
+    end
+
+    def possible_roles_permissions
+      PERMISSIONS-[get_level(user)]
+    end
+
+    def requested
+      AccessRequest.where(assessment_id: assessment.id)
```

Comment:
May want to remove references to assessment.  We will likely need to change it to inventory, but let's leave them out for now.